### ISSUE TITLE
Pressing enter without available items in `help_tags` leads to nil-value error

### ISSUE THREAD

**Issue by jonasstr** [nvim-telescope/telescope.nvim#1283]
### Description

When no items are available in `Telescope help_tags` and the user presses Enter, an error is shown instead of doing nothing or alternatively informing the user that there are no items available.

### Neovim version

NVIM v0.5.0
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3

### Operating system and version

Linux Mint 20.2

### checkhealth telescope

```markdown
health#telescope#check
========================================================================
## Checking for required plugins
  - OK: plenary installed.
  - OK: nvim-treesitter installed.

## Checking external dependencies
  - OK: rg: found ripgrep 11.0.2
  - WARNING: fd: not found. Install [sharkdp/fd](https://github.com/sharkdp/fd) for extended capabilities

## ===== Installed extensions =====
```

### Steps to reproduce

1. `nvim -nu minimal.lua`
2. Type the command `Telescope help_tags`
3. Enter a search term that leads to no items showing up, e.g. `aaaaaaaaaaaaa`
4. Press Enter.

### Expected behavior

Either don't close the window and show an info message
or close the window and show an info message.

In any case there should not be a `nil value` error.

### Actual behavior

The following error is shown:
```
E5108: Error executing lua .../start/telescope.nvim/lua/telescope/builtin/internal.lua:588: 
attempt to index local 'selection' (a nil value)
```
and the Telescope window is closed.

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvim/site]]
local package_root = '/tmp/nvim/site/pack'
local install_path = package_root .. '/packer/start/packer.nvim'

local function load_plugins()
    require('packer').startup {
        {
            'wbthomason/packer.nvim',
                {
                    'nvim-telescope/telescope.nvim',
                    requires = {
                        'nvim-lua/plenary.nvim',
                        { 'nvim-telescope/telescope-fzf-native.nvim', run = 'make' },
                    },
                },
                -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
        },
            config = {
                package_root = package_root,
                compile_path = install_path .. '/plugin/packer_compiled.lua',
                display = { non_interactive = true },
            },
    }
    end

_G.load_config = function()
    require('telescope').setup()
    -- ADD INIT.LUA SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
end

if vim.fn.isdirectory(install_path) == 0 then
    print("Installing Telescope and dependencies.")
    vim.fn.system { 'git', 'clone', '--depth=1', 'https://github.com/wbthomason/packer.nvim', install_path }
end

load_plugins()
require('packer').sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua load_config()]]
```

**Comment by Conni2461** [comment]
Thanks for reporting should be fixed

**Comment by jonasstr** [comment]
I just noticed this also occurs for `Telescope git_commits`, `git_bcommits`, `git_branches`, `git_stash`, `current_buffer_fuzzy_find`, `commands`, `colorscheme`, `planets`, `command_history`, `search_history`, `buffers`, `autocommands`, `filetypes`, `keymaps`, ...
I stopped here since this occurs for most of the commands (not all). I could submit a PR that fixes these if you want. Could there maybe be a variable with `[telescope] Nothing currently selected`  as a string so the text does not need to be repeated all the time?
```lua
if selection == nil then
    print "[telescope] Nothing currently selected"
    return
end
```
Like
```lua
if selection == nil then
    print no_current_selection_msg
    return
end
```
